include/grapphs/graph.h

```diff
@@ -55,6 +55,13 @@ namespace gpp {
          */
         virtual bool disconnect(index_type from, index_type to) = 0;
 
+        /**
+         * Returns true if there are not vertices in this graph
```

Comment:
Nitpick, there is a typo here: `there are **not** vertices`, could you fix please?